website/content/docs/release-notes/v0_9_0.mdx

```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+page_title: v0.9.0
+description: |-
+  Boundary release notes for v0.9.0
+---
+
+# [Boundary v0.9.0](https://www.boundaryproject.io/downloads)
+
+The release notes below contain information about new functionality available in the Boundary v0.9.0 release.
+To see a granular record of when each item was merged into the Boundary project, please refer to the [Changelog](https://github.com/hashicorp/boundary/blob/main/CHANGELOG.md).
+To learn about what Boundary consists of, we highly recommend you start at the [Getting Started Page](/docs/getting-started).
+
+Lastly, for instructions on how to upgrade an existing Boundary deployment to v0.9.0, please review Boundary's [general upgrade guide](https://learn.hashicorp.com/tutorials/boundary/upgrade-version).
+
+## Boundary v0.9.0 Highlights
+
+**HCP Boundary Public Beta:** Boundary is coming to the HashiCorp Cloud Platform! HCP Boundary provides an easy way to securely access 
+critical systems with fine-grained authorizations based on trusted identities. Boundary on HashiCorp Cloud Platform provides a fully managed, 
+single workflow to securely connect to hosts and critical systems across Kubernetes clusters, cloud service catalogs, and on-premises infrastructure.
```

Comment:
Do we want to highlight Kubernetes here, considering we have no native integration yet and it'd have to be done via manual configuration via e.g. static?